### ISSUE TITLE
Media: Add option to apply same texts to Caption and Alt text

### ISSUE THREAD

**Issue by kspilarski** [Automattic/wp-calypso#59718]
### What

In Media, add the option to apply or enable the use of same texts for Caption and Alt text. 

### Why

There are times when texts were added as captions, and alt text box is excluded and left blank. If the image file is missing, it won't show an alt text. 
There is also a user case when texts were intended as captions, but were added in alt text (the image added to a post didn't show the text meant as caption, as it was added as alt text mistakenly).

### How

Image from Media with Caption and Alt text: 

<img width="1062" alt="Screen Shot 2022-01-04 at 5 39 03 PM" src="https://user-images.githubusercontent.com/36863633/148096755-7fcd6e1f-6945-45a1-9911-bc1d8ac67388.png">

Reference: 4659272-zen
